### PR TITLE
[feature](statistics)Support analyze materialized view.

### PR DESCRIPTION
**Pull request by Jibing-Li**
While user analyze a whole table or auto analyze a table, collect the columns in materialized view as well.

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Jibing-Li** [comment]
run buildall

**Comment by Jibing-Li** [comment]
run buildall

**Comment by Jibing-Li** [comment]
run buildall

**Comment by Jibing-Li** [comment]
run buildall

**Comment by Jibing-Li** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/qe/ShowExecutor.java [hunk 1]
```diff
@@ -2502,43 +2502,54 @@ private void handleShowColumnStats() throws AnalysisException {
         ShowColumnStatsStmt showColumnStatsStmt = (ShowColumnStatsStmt) stmt;
         TableName tableName = showColumnStatsStmt.getTableName();
         TableIf tableIf = showColumnStatsStmt.getTable();
-        List<Pair<String, ColumnStatistic>> columnStatistics = new ArrayList<>();
+        List<Pair<Pair<String, String>, ColumnStatistic>> columnStatistics = new ArrayList<>();
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/qe/ShowExecutor.java:2505]: nit: add some comment to explain the meaning of inner pair

**Review by morrySnow — DISMISSED** [review]

**Comment by yiguolei** [comment]
run buildall

**Comment by Jibing-Li** [comment]
run buildall

**Review by morrySnow — APPROVED** [review]

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**